enchant.lua

```diff
@@ -694,13 +688,11 @@ function HandleEnchantCommand( Split, Player )
 		return true
 	end
 
-	local Result, ReturnMessage, Enchantment = EnchantItem( Split )
+	local ReturnMessage, ItemName, EnchantmentName, PlayerName = EnchantItem( Split )
 
-	if not Result then
-		SendMessage( Player, ReturnMessage )
-	else
-		SendMessage( Player, MessageSuccess )
-		local MsgString = string.format( LogMessageSuccess, Enchantment, ReturnMessage, Split[2] )
+	SendMessage( Player, ReturnMessage )
+	if ReturnMessage == MessageSuccess then
```

Comment:
This is bad choice. Prefer to keep the `IsSuccess` boolean return value from the `EnchantItem` function.